File: javaLoginCore/src/main/java/com/hamdan/forzenbook/java/login/core/domain/LoginEntrys.java
package com.hamdan.forzenbook.java.login.core.domain;

import androidx.annotation.NonNull;

import com.hamdan.forzenbook.java.core.Entry;

public class LoginEntrys {
    final private Entry email;
    final private Entry code;

    public LoginEntrys(Entry inEmail, Entry inCode) {
        email = inEmail;
        code = inCode;
    }

    @NonNull
    public Entry getEmail() {
        return email;
    }

    @NonNull
    public Entry getCode() {
        return code;
    }
}

File: loginCore/src/main/java/com/hamdan/forzenbook/login/core/domain/usecase/LoginEntrys.kt
package com.hamdan.forzenbook.login.core.domain.usecase

import com.hamdan.forzenbook.core.Entry
import com.hamdan.forzenbook.core.EntryError

data class LoginEntrys(
    val email: Entry = Entry("", EntryError.EmailError.None),
    val code: Entry = Entry("", EntryError.CodeError.None),
)
